charmstore.go

```diff
@@ -247,15 +247,11 @@ func (s *CharmStore) Resolve(ref *charm.Reference) (*charm.URL, error) {
 			case "":
 				etype = "entity"
```

Comment:
I'm thinking we should be consistent with other error messages here and use "charm or bundle". No need to change it in this PR though, just muttering to myself :)